packages/react-native-audio-api/common/cpp/audioapi/core/effects/ChannelMergerNode.cpp

```diff
@@ -0,0 +1,41 @@
+#include <audioapi/core/BaseAudioContext.h>
+#include <audioapi/core/effects/ChannelMergerNode.h>
+#include <audioapi/utils/AudioArray.h>
+#include <audioapi/utils/AudioBus.h>
+
+namespace audioapi {
+
+ChannelMergerNode::ChannelMergerNode(
+    BaseAudioContext *context,
+    unsigned int numberOfInputs)
+    : AudioNode(context, numberOfInputs) {
+  channelCount_ = 1;
+  channelCountMode_ = ChannelCountMode::EXPLICIT;
+  channelInterpretation_ = ChannelInterpretation::SPEAKERS;
+
+  outputBuses_.clear();
+  outputBuses_.push_back(
+      std::make_shared<AudioBus>(
+          RENDER_QUANTUM_SIZE, numberOfInputs_, context->getSampleRate()));
+
+  isInitialized_ = true;
+}
+
+void ChannelMergerNode::processNode(
+    const std::vector<std::shared_ptr<AudioBus>> &inputBuses,
+    int framesToProcess) {
+  auto outputBus = getOutputBus(0);
+
+  for (unsigned int i = 0; i < numberOfInputs_; ++i) {
+    bool isInputConnected = i < inputBuses.size() && inputBuses[i];
```

Comment:
It seems cursed you should always specify what are you checking implicit bool cast of `inputBuses[i]` here is an issue
https://clang.llvm.org/extra/clang-tidy/checks/readability/implicit-bool-conversion.html
```suggestion
  size_t count_ = std::min(numberOfInputs_, inputBuses.size()); // cache size
  for (size_t i = 0; i < count_; ++i) {
    bool isInputConnected = inputBuses[i] != nullptr;
```
this should do